src/api/java/com/minecolonies/api/util/BlockPosUtil.java

```diff
@@ -352,6 +352,22 @@ public static int getMaxDistance2D(@NotNull final BlockPos block1, @NotNull fina
         return Math.max(xDif, zDif);
     }
 
+    /**
+     * Gets the actual distance in blocks between two positions
+     *
+     * @param pos1 Blockpos 1
+     * @param pos2 Blockpos 2
+     * @return distance in blocks.
+     */
+    public static double getDistance(final BlockPos pos1, final BlockPos pos2)
```

Comment:
There is a method on the blockpos todo this.

Reply:
I thought that aswell, but was and still am unable to find any :D I only see a square one